SU-SWS/stanford_basic
# D8 core 941 margin top

# READY FOR REVIEW 

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-941_

# Needed Review By (Date)
- Sprint end


# Urgency
Needs to get done
 
# Steps to Test

- Checkout this branch
- npm run publish ; lando drush cr
- Verify Check all wysiwyg headings and styles with figma: https://www.figma.com/file/mQ5izx1G6f8DEUkgwWIHWP/D8-End-User-Display?node-id=0%3A1
Check Drop cap on both Chrome and Firefox

# Affected Projects or Products 
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# QA

## Cross-Browser testing:
What browsers have been tested?
- Chrome - Yes
- Firefox - Yes
- Safari
- Internet Explorer
- Android (Edge)
- iPhone (Safari)

## Unit Testing


# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-941

## Related PRs


## More Information

## Folks to notify
`@sherakama` 
`@kerri-augenstein`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
